server/index.js

```diff
@@ -11,52 +12,101 @@ app.use(express.urlencoded({ extended: true }));
 app.use(cors());
 app.use(express.static(path.join(__dirname, '..', 'dist')));
 
-app.get('/', (req, res) => {
-  console.log('GOT a GET');
+// ////////////////////////Get Requests////////////////////////////// //
+
+app.get('/products/dept/:dept', async (req, res) => {
+  helpers.getDept(req.params.dept, (err, results) => {
+    res.json(results);
+  });
 });
 
-app.get('/products/dept/:dept', (req, res) => {
-  let formattedDept = formatName(req.params.dept);
-  RecommendedItem.find({ department: formattedDept }, (err, results) => {
+app.get('/products/brand/:brandName', async (req, res) => {
+  helpers.getBrands(req.params.brandName, (err, results) => {
     res.json(results);
   });
 });
 
-app.get('/products/brand/:brandName', (req, res) => {
-  // escape certain characters from request url
-  let brandWords = req.params.brandName.split(/[,.\s-&amp]/);
-  RecommendedItem.find({
-    brand: {$regex: `^${brandWords.join('.*\s*')}$`, $options: 'i'}}, (err, results) => {
+app.get('/products/price/min=:minPrice&max=:maxPrice', async (req, res) => {
```

Comment:
I like your use of the params object here. Seems much cleaner and transparent than what I've been doing.